src/game.cpp

```diff
@@ -115,6 +115,38 @@ const skill_id skill_firstaid( "firstaid" );
 const species_id ZOMBIE( "ZOMBIE" );
 const species_id PLANT( "PLANT" );
 
+const efftype_id effect_adrenaline_mycus( "adrenaline_mycus" );
```

Comment:
It does not add, it just moves those to the top. However, I would be glad to extract them all into a separate file, but which?